<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageMethod</class>
 <widget class="QMainWindow" name="ManageMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>检测方法管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icon.qrc">
    <normaloff>:/images/clipboard.png</normaloff>:/images/clipboard.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="inputKeyword"/>
          </item>
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string>查询</string>
         </property>
         <widget class="QLineEdit" name="txtName">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>18</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSearch">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>18</y>
            <width>75</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>20</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>名称</string>
          </property>
         </widget>
         <widget class="QComboBox" name="inputSubject">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>17</y>
            <width>141</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="inputCategory">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>17</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>22</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>专业</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>20</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>亚专业</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Icon.qrc"/>
 </resources>
 <connections/>
</ui>
